editor/plugins/tile_map_editor_plugin.cpp

```diff
@@ -284,6 +308,15 @@ void TileMapEditor::_update_palette() {
 
 	palette->set_fixed_icon_size(Size2(min_size, min_size));
 	palette->set_fixed_column_width(min_size * MAX(size_slider->get_value(), 1));
+	palette->set_same_column_width(true);
+	manual_palette->set_fixed_icon_size(Size2(min_size, min_size));
+	manual_palette->set_same_column_width(true);
+
+	List<int> tiles;
```

Comment:
is there any reason to move this line here from L269? it's hard to tell just looking at the diff

Reply:
There's not really a reason, it's left over from when I was initially going to have the autotile select panel appear in the same panel as the regular palette panel. From a grouping standpoint is does make sense to have this block of code next to where its results are needed. It could be put back to where it was before without issue though, and that might be preferable for the early return. The performance gain from this would be basically negligible though.